### PR TITLE
jvgrep 5.8.1

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/jvgrep.rb [hunk 1]
```diff
@@ -3,8 +3,9 @@
 class Jvgrep < Formula
   desc "Grep for Japanese users of Vim"
   homepage "https://github.com/mattn/jvgrep"
-  url "https://github.com/mattn/jvgrep/archive/v5.8.0.tar.gz"
-  sha256 "5a31dbe457ba5ed8492a0ac59eecb22356754f456086ee4e95f296fc6f4bc8c8"
+  url "https://github.com/mattn/jvgrep.git",
```
Comment (javian) [inline Formula/jvgrep.rb]: Why was this change required ?

Comment (chenrui333) [reply Formula/jvgrep.rb]: Oh, I am actually trying to consolidate the go-lang formula to use tag instead of sha hash whenever I did the upgrades.

**Comment by chenrui333** [comment]
Reverted to tarballs download.
